alternator/ttl.cc

```diff
@@ -441,21 +447,35 @@ future<> expiration_service::run() {
             }
         }
         // FIXME: replace this silly 1-second sleep by something smarter.
```

Comment:
What are we going to do about this?

Reply:
The plan is to have sort of "controller" which slows down the scan to the pace we need it to be, based on a parameter of expiration delay. If the user configures expiration delay of 48 hours, and we're scanning too fast and will finish in 2 hours, we should slow down the scan. On the other hand, for tests, I want the scan to run all the time, for the test to finish quickly.
But I wanted to fix this later.